docs-chef-io/content/supermarket/supermarket_upgrade.md

```diff
@@ -11,205 +11,244 @@ gh_repo = "supermarket"
+1. Stop the Supermarket services:
 
-        ```bash
-        dpkg -i /path/to/package/supermarket*.deb
-        ```
-
-      - For RHEL / CentOS:
+    ```bash
+    sudo supermarket-ctl stop
+    ```
 
-        ```bash
-        rpm -Uvh /path/to/package/supermarket*.rpm
-        ```
+1. Back up the `/var/opt/supermarket` directory.
+1. Download a Supermarket package from [Chef Downloads](https://www.chef.io/downloads).
+1. Install the new package using your distribution's package manager:
 
-  1. Determine the current installed PG (PostgreSQL) version which comes bundled with supermarket. This step will determine if we need to run the next step. Run the below command:
+    - For Ubuntu:
 
       ```bash
-      sudo /opt/supermarket/embedded/bin/postgres --version
+      dpkg -i /path/to/package/supermarket*.deb
       ```
 
-      If in the new version of supermarket the PG version is a major upgrade from the current installed PG version then skip the next step of starting supermarket. For details on which version of supermarket installs which version of PG please refer this section: [Supermarket Postgres Version Matrix](#supermarket-postgres-version-matrix)
-
-  1. Start the Chef Supermarket services (skip this step if there is any major version upgrade for PostgreSQL in the new version of supermarket):
+    - For RHEL:
 
       ```bash
-      sudo supermarket-ctl start
+      rpm -Uvh /path/to/package/supermarket*.rpm
       ```
 
-  1. Reconfigure Chef Supermarket server:
+1. Start the Chef Supermarket services:
 
-      ```bash
-      sudo supermarket-ctl reconfigure
-      ```
+    ```bash
+    sudo supermarket-ctl start
+    ```
 
-  1. Once the Private Supermarket upgrade finishes, restart the services that run Chef Supermarket to clear the old installation of Chef Supermarket from the server memory.
+1. Reconfigure Chef Supermarket server:
 
-      ```bash
-      systemctl list-units | grep runsvdir
-      ```
+    ```bash
+    sudo supermarket-ctl reconfigure
+    ```
 
-  1. Restart the unit:
+1. After the upgrade finishes, restart the services that run Chef Supermarket to clear the old installation from server memory.
 
-      ```bash
-      systemctl restart UNIT_NAME
-      ```
+    ```bash
+    systemctl list-units | grep runsvdir
+    ```
+
+1. Restart the unit:
 
-      This will restart the `runsvdir`, `runsv`, and `svlogd` service processes that run Chef Supermarket.
+    ```bash
+    systemctl restart <UNIT_NAME>
+    ```
 
-## Release Specific Upgrade: Supermarket 5.0 and PostgreSQL 13.4
+    This restarts the `runsvdir`, `runsv`, and `svlogd` service processes that run Chef Supermarket.
 
-Supermarket 5.0 upgrades PostgreSQL from 9.3 to 13.4. The 5.0 upgrade process requires a one-time downtime to vacuum, upgrade, and re-index the database.
+## Upgrade to Supermarket 4.2.x to 5.x
 
-### Supermarket 5.0 Changes
+When you upgrade from Supermarket 4.2.x to 5.x, this also upgrades PostgreSQL from 9.3 to 13.4.
```

Comment:
only till 5.2 we have PG 13.4. But with this release we will have PG 13.18. So this statement needs to be corrected.